src/components/presentational/SingleDataPoint/SingleDataPoint.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { CSSProperties } from 'react';
+import './SingleDataPoint.css';
+import { UnstyledButton } from '..';
+import { ColorDefinition } from '../../../helpers/colors';
+
+export interface SingleDataPointProps {
+    label: string;
+    statusText?: string;
+    statusColor?: ColorDefinition;
+    value?: number;
+    units?: string;
+    onClick?: () => void;
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: SingleDataPointProps) {
+    const getContents = () => {
+        return <>
+            <div className="mdhui-single-data-point-value-units">
+                {!props.value &&
+                    <div className="mdhui-single-data-point-value">...</div>
+                }
+                {props.value &&
+                    <div className="mdhui-single-data-point-value">{props.value.toLocaleString()}</div>
+                }
+                {props.value && props.units &&
+                    <div className="mdhui-single-data-point-units">{props.units}</div>
+                }
+            </div>
+            <div className="mdhui-single-data-point-label">{props.label}</div>
+            {props.statusText &&
+                <div className="mdhui-single-data-point-status" style={props.statusColor ? {color: props.statusColor} as CSSProperties : undefined}>{props.statusText}</div>
+            }
+        </>;
+    };
+
+    return <div ref={props.innerRef}>
```

Comment:
This pattern makes me wonder if we should move this into UnstyledButton...e.g. UnstyledButton becomes a div if there is no click handler.  Probably gets complicated with the refs though and we'd need a div wrapper or something which might mess stuff up I guess

Reply:
Yeah.  I'm also not sure when you would use an `UnstyledButton` without a click listener (it is not currently optional).

It might be nice to have an "MDH-UI div" component that renders as an `UnstyledButton` when a click listener is provided, else a `div`.  That could be used anywhere you want to use a regular `div`, and if you happen to add a click listener to one, you don't have to think about reworking it for the focus traversal stuff.